qutip/quantum_systems/jaynes_cummings.py

```diff
@@ -72,7 +72,7 @@ def jaynes_cummings(
     # Atomic operators (tensor with cavity system)
     operators['sigma_minus'] = tensor(qeye(n_cavity), destroy(2))
     operators['sigma_plus'] = operators['sigma_minus'].dag()
-    operators['sigma_z'] = tensor(qeye(n_cavity), sigmaz())
+    operators['sigma_z'] = operators['sigma_plus'] * operators['sigma_minus'] - operators['sigma_minus'] * operators['sigma_plus']
     operators['sigma_x'] = tensor(qeye(n_cavity), sigmax())
     operators['sigma_y'] = tensor(qeye(n_cavity), sigmay())
```

Comment:
sorry, i wasn't explicit enough in my previous comment; i meant to check all of the pauli operators.  sigma_x is invariant under this redefinition, but sigma_y probably needs a complex conjugate?

Reply:
Yeah, just could also remove those two, and make sure how the model is defined is just explained in the docs.  Either is fine, upto you!